### ISSUE TITLE
Cannot upload images by drag&drop or by pasting from clipboard

### ISSUE THREAD

**Issue by alexandrudima** [Microsoft/vscode-pull-request-github#320]
In the web UI of GitHub, it is possible to upload images, while this is not possible with this extension.

This is sometimes useful when discussing how things look like.

**Comment by connorshea** [comment]
IIRC the GitHub API doesn't provide any way to upload images >.<

**Comment by smith** [comment]
@connorshea in the GitHub web app if you drop an image it creates a markdown link and uploads the image to user-images.githubusercontent.com. I guess you would have to implement that behavior in the extension's text fields in order for that to work.

That's a bummer too, because if I can't drag images from Gif Keyboard to PR approvals, I can't approve PRs with this extension. It's great for everything else though!

**Comment by mothepro** [comment]
_Workaround idea_: A (potentially separate) plugin that uploads images that are DnD to another online service i.e. Imgur.
Then use the link generated from that service inside the markdown of the comment/description.

**Comment by Spookywy** [comment]
Hey 👋 Is there a roadmap for enabling the addition of images directly through the Visual Studio Code extension in the near future?
That would be a nice improvement!

**Comment by jamesdongdong** [comment]
I am looking forward to this feature!

**Comment by ansh-les** [comment]
Yes please this would be so useful while reviewing frontend PRs.

**Comment by nezzard** [comment]
No updates?

**Comment by wslp12** [comment]
It is using GitHub for enterprises.
When I upload an image when I create pr, it seems to be saved somewhere with my nickname as uuid value

I don't know if this feature is offered as an api, but as far as I know, these api are not available at the moment. I really wish this extension would provide this feature.

**Comment by SeriousHoax** [comment]
Please add this feature 🙏

**Comment by cm3cm** [comment]
Would love to have this!

**Comment by makrasnov100** [comment]
+1 uploading images directly with the extension inside PR description and comments, especially pasting in a screenshot, is the only thing holding me back from switching from the web UI
